migration/migrator/migrations/course/20210622144445_team_names.py

```diff
@@ -0,0 +1,33 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("""ALTER TABLE gradeable_teams ADD COLUMN team_name character varying(255) DEFAULT NULL;""")
```

Comment:
Probably want a `IF NOT EXISTS` after the `ADD COLUMN` incase we run this after a rollback